packages/base-dao/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@venncity/base-dao",
+  "version": "1.0.0",
+  "author": "Venn Engineering",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest",
+    "coverage": "jest --coverage",
+    "test:watch": "npm run test -- --watchAll",
+    "lint": "npx eslint --fix src",
+    "test:ci": "jest --ci --reporters=default --reporters=jest-junit --runInBand --coverage",
+    "lint:ci": "npx eslint src --format junit -o $LINT_JUNIT_OUTPUT",
+    "generate-opencrud-schema": "node ./scripts/generateOpenCRUDSchema.js"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "npm run lint",
+      "git add"
+    ]
+  },
+  "dependencies": {
+    "@venncity/cascade-delete": "^1.0.0",
+    "@venncity/errors": "^1.4.1",
+    "@venncity/graphql-transformers": "^1.0.0",
+    "@venncity/nested-config": "^1.2.6",
+    "@venncity/opencrud-schema-provider": "^1.2.0",
+    "@venncity/sequelize-data-provider": "^1.0.0",
+    "@venncity/sequelize-model": "^1.1.0",
+    "async": "^3.1.0",
+    "dataloader": "^1.4.0",
+    "lodash": "^4.17.15",
+    "sequelize": "^5.21.2",
+    "pluralize": "^8.0.0"
+  },
+  "devDependencies": {
+    "@venncity/auth": "^6.3.0",
```

Comment:
auth and event pub sub are dev dependencies?

Reply:
yes, they are here only so the rest of the code can be tested easily and later will move to the consuming code, e.g. persistency